src/base/model.js

```diff
@@ -633,6 +633,9 @@ ModelBase.prototype.getTimestampKeys = function() {
  * @param {string} [options.method]
  *   Either `'insert'` or `'update'` to specify what kind of save the attribute
  *   update is for.
+ * @param {string} [options.date]
+ *   Either a Date object or ms since the epoch. Specify what date is used for
+ *   the timestamps updated.
```

Comment:
Can you change this a little bit to make it clearer what the purpose of `date` is? Something like:

> Specify what date is used as the value of `now` when creating or updating timestamp attributes. This overrides the current date with the specified value.